recipes/omniscidb-cuda/recipe/meta.yaml

```
@@ -0,0 +1,97 @@
+{% set name = "omniscidb-cuda" %}
+{% set version = "4.8.1" %}
+{% set sha256 = "300cdbbaa9108879f7ec22d9cf6210ca2700a3ef99c356a5c00fbf3aeb3a4713" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    Skip-installing-Boost-libs.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
+    - boost-cpp 1.68.*        # to avoid cmake picking up boost-cpp 1.70+
+    - cmake
+    - make
+    - go
```

Typically one would use the following. Is there a reason not to here?

```suggestion
    - {{ compiler('cgo') }}
```

---

This may fix the Go issues you are running into. So please check when you have a chance. 🙂

---

This gives:
```
conda.exceptions.ResolvePackageNotFound: 
  - cgo_linux-64
```

---

Also you will need to add [this file]( https://github.com/conda-forge/singularity-feedstock/blob/b8d8d6dee800800ff041ef6d0d0868d8d3d3863d/recipe/conda_build_config.yaml ) to your recipe.

---

yes, that helped, thanks!
Next, for a definite answer if using `{{ compiler('cgo') }}` resolves the `go-CC` issue, I'll need to disable the local `CONDA_ENV_SYSROOT` change in `install_nvcc.sh` script..

---

Great! Please let us know what you find 🙂

---

I can confirm that omnisci-cuda now builds with the nvcc (PR 13) and go.

---

I'll double-check with the latest changes in https://github.com/conda-forge/nvcc-feedstock/pull/13 .

---

Glad to hear it! 😄